currency.js

```diff
@@ -0,0 +1,61 @@
+var https = require('https')
+
+function convertCurrency(amount, fromCurrency, toCurrency, cb) {
+    var apiKey = '63a107cf0f91eaa9a44f'
```

Comment:
i think this apikey should probably be an environment variable in the `.env`, can you remove this string and add it to the environment variables, like the other one?